src/parser.c

```diff
@@ -958,7 +958,7 @@ network parse_network_cfg_custom(char *filename, int batch, int time_steps)
 #endif
         }
         else if (lt == EMPTY) {
-            layer empty_layer;
+            layer empty_layer={0};
```

Comment:
Use
` layer empty_layer = { (LAYER_TYPE)0};`

Otherwise it causes an error when compiling in C ++ mode:

> /Users/travis/build/AlexeyAB/darknet/src/parser.c:967:32: fatal error: cannot initialize a member subobject of type 'LAYER_TYPE' with an rvalue of type 'int'
>             layer empty_layer={0};
>                                ^
> 1 error generated.

Reply:
fixed